CONTRIBUTING.md

```diff
@@ -331,6 +344,25 @@ xgettext --keyword=_ --language=C --add-comments -o po/labwc.pot src/menu/menu.c
 
 [See this tutorial for further guidance](https://www.labri.fr/perso/fleury/posts/programming/a-quick-gettext-tutorial.html)
 
+# Upversion
+
+It is generally only the lead-maintainer who will upversion, but in order
+not to forget any key step or in case someone else needs to do it, here
+follow the steps to be taken:
+
+1. If appropriate, update `revision` in `subprojects/wlroots.wrap` and run
+   `git commit -m 'wlroots.wrap: use A.B.C'`
+2. Update `NEWS.md` with the release details and run
+   `git commit -m 'NEWS.md: update notes on X.Y.Z'`
+3. In `meson.build` update the version and (if required) the wlroots
+   dependency version. Then run `git commit -m 'build: bump version to X.Y.Z'`
+4. Run `tag -a X.Y.Z`. The first line of the commit message should be
```

Comment:
I assume this should be `git tag`.

Other than that, LGTM

Reply:
Whoops. Thanks. Will fix.